css/components/guide-nav.css

```diff
@@ -24,3 +24,42 @@
   display: inline;
   font-weight: bold;
 }
+
+.lgd-guide-nav__list-item--active {
+  font-weight: bold;
+}
+
+.lgd-guide-nav__list-item > a {
+  font-weight: normal;
+}
+
+.block-localgov-guides-contents {
+  margin-top: var(--spacing-largest);
+  margin-bottom: var(--spacing-largest);
+}
+
+.lgd-prev-next__link {
+  flex-wrap: wrap;
+  padding: var(--spacing-smaller);
+  text-decoration: none;
+}
```

Comment:
Can you check that this `flex-wrap` works? Without a `display: flex`, I'm not sure it will.

Reply:
@alexburrows , just discussing at Merge Monday and @markconroy just wanted to check on the flex-wrap question.